src/index.md

```diff
@@ -8,6 +8,93 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+build web apps that work offline.
+
+Modern offline web apps are possible thanks to **service workers** (SWs). SWs
+are background workers, written in JavaScript, that can locally save all of
```

Comment:
👍 for contextually linking out to MDN docs for some of these concepts newcomers might wish to dig into further.